db/migrate/20151110150858_add_content_id_to_redirect.rb

```diff
@@ -0,0 +1,12 @@
+class AddContentIdToRedirect < ActiveRecord::Migration
+  def up
+    add_column :newest_redirects, :content_id, :string
+    execute 'UPDATE newest_redirects SET content_id = UUID()'
```

Comment:
We can't use SQL `UUID()` for this because the command is replicated down to the secondary servers in the cluster, and they generate their own UUIDs which are different to the primary (and to each other).

We need to generate these in Ruby so that they all get sent along statically through the replication tree.